File: app/src/main/java/com/health/openworkout/gui/preference/SoundPreferences.java
package com.health.openworkout.gui.preference;

import android.os.Bundle;

import androidx.preference.PreferenceFragmentCompat;

import com.health.openworkout.R;

public class SoundPreferences extends PreferenceFragmentCompat {

    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        setPreferencesFromResource(R.xml.sound_preferences, rootKey);
    }

}

File: app/src/main/java/com/health/openworkout/gui/preference/SoundPreferences.kt
package com.health.openworkout.gui.preference

import android.os.Bundle

import androidx.preference.PreferenceFragmentCompat

import com.health.openworkout.R

class SoundPreferences : PreferenceFragmentCompat() {
    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        setPreferencesFromResource(R.xml.sound_preferences, rootKey)
    }
}
